README.md

```diff
@@ -1,21 +1,37 @@
-# Project Name
+# Nutrition label
 
-> One paragraph statement about the project.
+> CSS/HTML Nutrition label template used for food labeling.
```

Comment:
Are you sure this is a template
lets give a more detailed desription

Reply:
OK

Reply:
OK